app/assets/javascripts/establishment_shot/charts.js

```diff
@@ -26,19 +23,70 @@ EstablishmentShot.Charts = (function () {
             ],
             [
-                '#DDDDDE',
                 '#6E3E00',
                 '#B05E00',
                 '#FCB100',
                 '#006FB0',
                 '#00A0FC'
             ]
         ],
+        queries = [
+            [
+                { key: 'co2_sheet_buildings_households_space_heating_cooling_co2_emissions' },
+                { key: 'co2_sheet_buildings_households_hot_water_co2_emissions' },
+                { key: 'co2_sheet_buildings_households_cooking_co2_emissions' },
+                { key: 'co2_sheet_buildings_households_appliances_light_co2_emissions' }
+            ],
+            [
+                { key: 'co2_sheet_industry_metal_co2_emissions' },
+                { key: 'co2_sheet_industry_chemical_co2_emissions' },
+                { key: 'co2_sheet_industry_food_co2_emissions' },
+                { key: 'co2_sheet_industry_paper_co2_emissions' },
+                { key: 'co2_sheet_energy_sector_total_co2_emissions' },
+                { key: 'co2_sheet_industry_other_co2_emissions' }
+            ],
+            [
+                { key: 'co2_sheet_agriculture_heat_co2_emissions' },
+                { key: 'co2_sheet_agriculture_power_and_light_co2_emissions' }
+            ],
+            [
+                { key: 'co2_sheet_transport_total_domestic_aviation_co2_emissions' },
+                { key: 'co2_sheet_transport_total_domestic_freight_co2_emissions' },
+                { key: 'co2_sheet_transport_total_private_transport_co2_emissions' },
+                { key: 'co2_sheet_transport_total_public_transport_co2_emissions' }
+            ]
+        ],
+        nonEnergyColor =  '#DDDDDE',
+        nonEnergyQueries = [
+            { key: 'co2_sheet_non_energy_emissions_built_environment' },
+            { key: 'co2_sheet_non_energy_emissions_industry_energy' },
+            { key: 'co2_sheet_non_energy_emissions_agriculture' },
+            { key: 'co2_sheet_non_energy_emissions_transport' }
+        ],
+        total_chart_attributes = {
+            series: [
+                        { key: 'co2_sheet_agriculture_total_co2_emissions_only_energetic',
+                          title: 'co2_sheet_agriculture_total_co2_emissions',
+                          fa_icon: '\uf06c' },
+                        { key: 'co2_sheet_industry_energy_total_co2_emissions_only_energetic',
+                          title: 'co2_sheet_industry_energy_total_co2_emissions',
+                          fa_icon: '\uf275' },
+                        { key: 'co2_sheet_transport_total_co2_emissions_only_energetic',
+                          title: 'co2_sheet_transport_total_co2_emissions',
+                          fa_icon: '\uf1b9' },
+                        { key: 'co2_sheet_buildings_households_total_co2_emissions_only_energetic',
+                          title: 'co2_sheet_buildings_households_total_co2_emissions',
+                          fa_icon: '\uf015' }
+                    ],
```

Comment:
No biggie, but the indenting is a bit off here.

Reply:
Thanks! I fixed the indent issue and will merge the PR as one commit when all checks have completed.